spark/src/test/java/org/apache/iceberg/actions/TestRemoveOrphanFilesAction.java

```diff
@@ -569,4 +663,220 @@ public void testRemoveOrphanFilesWithHadoopCatalog() throws InterruptedException
         .collectAsList();
     Assert.assertEquals("Rows must match", records, actualRecords);
   }
+
+  @Test
+  public void testFindOrphanFilesWithSameAuthority() throws Exception {
+    executeTest(
+        HDFS_SERVICENAME_USER_LOG_DATA_DUMMY_FILE,
```

Comment:
Makes me really want to do that Junit5 upgrade :) But this is better than before 👍